apollo-testing-support/src/commonMain/kotlin/com/apollographql/apollo3/testing/internal/runTest.kt

```diff
@@ -12,36 +11,10 @@ import kotlin.coroutines.EmptyCoroutineContext
  * When [skipDelays] is `true`, the block is executed in [kotlinx.coroutines.test.runTest], otherwise in `runBlocking`.
  */
 @ApolloInternal
-@OptIn(ExperimentalCoroutinesApi::class)
-fun runTest(
+expect fun runTest(
     skipDelays: Boolean = false,
     context: CoroutineContext = EmptyCoroutineContext,
     before: suspend CoroutineScope.() -> Unit = {},
     after: suspend CoroutineScope.() -> Unit = {},
     block: suspend CoroutineScope.() -> Unit,
-) {
-  if (skipDelays) {
-    kotlinx.coroutines.test.runTest(context) {
-      before()
-      try {
-        block()
-      } finally {
-        after()
-      }
-    }
-  } else {
-    runBlockingOrPromise(context) {
-      before()
-      try {
-        block()
-      } finally {
-        after()
-      }
-    }
-  }
-}
```

Comment:
Could you make that function return `Any` instead ? (`Unit` on JVM/native and `Promise<T>` on JS)

Reply:
It looks like JUnit doesn't like it, it wants the tests methods to return `void` (so we could make it work with `fun test() { runTest {} }` instead of `fun test() = runTest {}` but that's a bit annoying

Reply:
👍 sounds good